src/components/shared/Accordion.css

```diff
@@ -75,6 +75,7 @@
 .accordion ._content {
   border-bottom: 1px solid var(--theme-splitter-color);
   font-size: var(--theme-body-font-size);
+  overflow: auto;
```

Comment:
We still need to remove this `overflow: auto` here.
Everything else looks great.

Reply:
Sorry! I forgot about that one. Thanks!